app/templates/client/app/scripts/controllers/login.js

```diff
@@ -8,7 +8,14 @@
  * Controller of the clientApp
  */
 angular.module('clientApp')
-  .controller('LoginCtrl', function ($scope, Auth, $state) {
+  .controller('LoginCtrl', function ($scope, Auth, $state, Account, accessToken) {
+
+    // Try to verify the email when access token is being passed.
```

Comment:
@ordavidil 
* Set the access_token using the ```Auth.setAccessToken($stateParams.accessToken)```
* Please move this to the`state` resolve method and return the result of ```Account.verifyEmail()```.
```js
// Try to verify the email when access token is being passed.
if ($state.current.name == 'verifyEmail' && accessToken) {
Account.verifyEmail(accessToken).then(function () {
$scope.emailVerified = true;
 });
 }
```